.gitignore

```diff
@@ -1,2 +1,4 @@
+dist/
```

Comment:
We need `dist/` becase there's the option to require the package directly from the browser.